cmd/create.go

```diff
@@ -47,15 +75,73 @@ func executeCreate(devPath string) error {
 
 	dev := linguist.GetDevConfig(languagesDiscovered[0])
 	dev.Swap.Deployment.Name = filepath.Base(root)
+
+	var env string
+	if languagesDiscovered[0] == "unrecognized" {
+		fmt.Printf("Couldn't detect any language in your source. Recommended development environment: %s", log.BlueString(dev.Swap.Deployment.Image))
```

Comment:
I feel the need for the word `image`here. Maybe `Recommended image for development:`